imports/plugins/core/graphql/server/no-meteor/resolvers/xforms/catalogProduct.js

```diff
@@ -17,19 +18,20 @@ export const encodeCatalogProductOpaqueId = encodeOpaqueId(namespaces.CatalogPro
 export function xformProductMedia(mediaItem) {
   if (!mediaItem) return null;
 
-  const { metadata, large, medium, image, small, thumbnail } = mediaItem;
-
+  const { priority, toGrid, productId, variantId, URLs: { large, medium, original, small, thumbnail } } = mediaItem;
+  const absoluteUrl = Reaction.absoluteUrl().slice(0, -1);
```

Comment:
@aldeed Is this all that's needed? How can I take into account when a CDN is being used?